sync.go

```diff
@@ -307,6 +307,10 @@ func (lw *LibWallet) CancelSync() {
 
 	lw.walletLoader.SetNetworkBackend(nil)
 	loadedWallet.SetNetworkBackend(nil)
+
+	log.Info("Waiting to lose all peers")
+	peersWG.Wait()
```

Comment:
`CancelSync` can be called at other times without a need to wait for all peers to be lost. Maybe add a boolean `losePeers` parameter to control this function.